src/Lykke.MatchingEngine.Connector.Abstractions/Models/FeeSizeType.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Lykke.MatchingEngine.Connector.Abstractions.Models
+{
+    public enum FeeSizeType
+    {
```

Comment:
May be PascalCase? Seems like we are using 'int' representation of this enums for external contracts, we can use C#-style for sure.

Reply:
It's for backward compatibility with ME.